src/click/exceptions.py

```diff
@@ -134,17 +144,28 @@ def format_message(self):
             msg_extra = self.param.type.get_missing_message(self.param)
             if msg_extra:
                 if msg:
-                    msg += f".  {msg_extra}"
+                    msg += f". {msg_extra}"
```

Comment:
I don't think this should be changed. It's common for commandline tools (and other plaintext-based systems) to have two spaces after a colon.

Reply:
It's been inconsistent across all the projects, I've been standardizing on single space after period for years now. This is the only remaining case of two spaces after a period in Click (excluding old docs and comments).